experiments/calibration/README.md

```diff
@@ -0,0 +1,120 @@
+# Calibration of land model on Derecho
+
+This folder contains scripts to calibrate a land model (bucket or full land) on Derecho HPC.
+
+The folders are organized as such:
+
+```
+.
+├── global_bucket
+│   ├── calibrate_bucket.jl
+│   ├── forward_model.jl
+│   └── observation_map.jl
+├── global_land
+│   ├── calibrate_land.jl
+│   ├── forward_model.jl
+│   └── observation_map.jl
+├── job_scripts
+│   ├── calibrate_bucket.pbs
+│   └── calibrate_land.pbs
+└── shared
+    ├── make_training_locations.jl
+    ├── observation_data_global.jl
+    ├── observation_data_locations.jl
+    └── rand_locations.jl
+```
+
+## calibrate\_bucket.jl and calibrate\_land.jl
+
+Those scripts are the main script, calling the others. They get the arguments of the ClimaCalibrate function call
+for calibration
+
+```julia
+CAL.calibrate(
+    CAL.WorkerBackend,
+    ensemble_size,
+    n_iterations,
+    observations,
+    noise,
+    prior,
+    caldir,
+)
+```
+
+### forward\_model
+
+to work, a `forward_model(iteration, member)` function needs to be defined (in forward\_model.jl), this will run
+the model (bucket or land) and save the diagnostics.
+
+### observation\_map
+
+Furthermore, an `observation_map(iteration)` function needs to be defined (in observation\_map.jl), this will
+generate your loss from the model output, in the same format as your observations from data, contained in shared/observation\_data.
+For each member (parameters sample), a vector will be created from the function `process_member_data(root_path)` located in the script
+observation\_map.jl.
```

Comment:
```suggestion
`observation_map.jl` defines `observation_map(iteration)`, a function that generate the loss from the model output. The loss function has to have the same format as your observations from data, contained in `shared/observation_data.jl`.
For each member (parameters sample), a vector will be created from the function `process_member_data(root_path)` located in the script
`observation_map.jl`.
```

Reply:
Is `observation_map` the observation operator `H` (mapping model output `x` to observables `y`) or generating the loss function (i.e., `norm(y - H(x))` where `norm` is a suitable weighted norm)? Please be clear here.